buildah.go

```diff
@@ -19,9 +19,17 @@ const (
 	// identify working containers.
 	Package = "buildah"
 	// Version for the Package
-	Version       = "0.1"
+	Version = "0.1"
+	// The value we use to identify what type of information, currently a
+	// serialized Builder structure, we are using as per-container state.
+	// This should only be changed when we make incompatible changes to
+	// that data structure, as it's used to distinguish containers which
+	// are "ours" from ones that aren't.
 	containerType = Package + " 0.0.1"
```

Comment:
IDK as I'm flip fllopping, so I'll just ask.   Should/do we need to state the basis of the Builder structure?  i.e. based on Docker's Builder structure or whatever?  Do we want to identify who "They" is somewhere in here?  I'm getting at this as we might make incompatible changes, but isn't it likely "They" might too?